task-maker-async/src/store.rs

```diff
@@ -0,0 +1,98 @@
+#![allow(dead_code)]
+
+use serde::{Deserialize, Serialize};
+
+type HashData = Vec<u8>;
```

Comment:
I'm not sure what you mean here - but I believe changing the array size should always be fine.

I used Vec<u8> because that's what we use today for file hashes ;)